src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiersTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks.processor;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.ByteArrayInputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipInputStream;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ZipVerifiersTest {
+
+    private String publicKeyBase64 =
+        "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDEXfjyDFFigzsmFvTe2cWZ45ggH/XoS/3C6Ur/"
+            + "V0egi8k5hnIIgPEOUqhrX5UcQorSX7bIlMped6TtPkYdGs/QI6S5m2uz+6Mjai7ZfACGhYxIs8"
+            + "35msqvRsDM0tIle/h3eZJb7iPE0anMWb8MkBYU3D3vAnPdBZxiEIwNMUNzqQIDAQAB";
+    
+    @Test
+    public void should_verify_signed_file_successfully() throws Exception {
+        byte[] test1PdfBytes = toByteArray(getResource("test1.pdf"));
+        byte[] test1SigPdfBytes = toByteArray(getResource("signature/test1.pdf.sig"));
+        ZipVerifiers.verifySignature(publicKeyBase64, test1PdfBytes, test1SigPdfBytes);
+    }
+
+    @Test
+    public void should_not_verify_other_file_successfully() throws Exception {
+        byte[] test2PdfBytes = toByteArray(getResource("test2.pdf"));
+        byte[] test1SigPdfBytes = toByteArray(getResource("signature/test1.pdf.sig"));
+        ZipVerifiers.verifySignature(publicKeyBase64, test2PdfBytes, test1SigPdfBytes);
+    }
+
+    @Test
+    public void should_verify_2_valid_filenames_successfully() throws Exception {
+        Map<String, byte[]> files = new HashMap<>();
+        files.put("documents.zip", new byte[0]);
+        files.put("signature.sig", new byte[0]);
+        assertThat(ZipVerifiers.verifyFileNames(files)).isTrue();
+    }
+
+    @Test
+    public void should_not_verify_more_than_2_files_successfully() throws Exception {
```

Comment:
it should be something more specific. something like `should_throw_file_content_verification_failed_when_zip_contains_multiple_sig_files`

Reply:
I think your version is overly long and not more meaningful, so on this we'll have to disagree ;-)

Reply:
`should_not_verify_more_than_2_files_successfully` doesn't gives correct message what 2 files you are talking about. are u saying 2 pdf's or 2 signature ?

Reply:
Any number of files in excess of 2 is a spec violation.